src/main/java/dinkplugin/notifiers/DiaryNotifier.java

```diff
@@ -142,6 +143,7 @@ private void initCompleted() {
                 diaryCompletionById.put(id, value);
             }
         }
+        log.debug("Finished initializing current diary completions: {} out of {}", getTotalCompleted(), diaryCompletionById.size());
```

Comment:
Think we could get away with having this be a `log.info`? It should only print once per log in/hop